src/components/Proposals/InfoBoxes.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Box,
+  Help,
+  IconTime,
+  Timer,
+  textStyle,
+  useLayout,
+  useTheme,
+  GU,
+} from '@aragon/ui'
+import { dateFormat, toMs } from '../../utils/date-utils'
+import { round, safeDiv } from '../../utils/math-utils'
+import SummaryBar from './SummaryBar'
+
+function InfoBoxes({ vote, setting }) {
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+
+  const totalVotes = parseFloat(vote.yeas) + parseFloat(vote.nays)
+  const support = safeDiv(parseFloat(vote.yeas), totalVotes)
+  const quorumProgress = safeDiv(vote.yeas, vote.votingPower)
```

Comment:
Looking very forward to avoid having to do this kind of calculation once https://github.com/aragon/token-amount/issues/5 is available :)!